README.md

```diff
@@ -94,7 +94,9 @@ the `tutorials` folder.
 <details>
   <summary> Training and reproduction </summary>
 
-the folder `training` contains a SpaCy project which will allow for reproduction of the results. This folder also includes the evaluation metrics on DaNE and scripts for downloading the required data. For more information, please see the training [readme](training/readme.md).
+The folder `training` contains a SpaCy project which will allow for reproduction of the results. This folder also includes the evaluation metrics on DaNE and scripts for downloading the required data. For more information, please see the training [readme](training/readme.md).
+
+Similarly, the folder `ner_extended`contains a spaCy project which will allow for reproduction of the results and models with fine-grained NER, using the DANSK dataset annotations. More information available in the [readme](ner_extended/README.md).
```

Comment:
can you change to folder structure to be `training/ner_extended` instead.

Reply:
Yes - also changed the named to ner_fine_grained to match the name it is referred to.